actionpack/test/controller/required_params_test.rb

```diff
@@ -48,4 +48,20 @@ class ParametersRequireTest < ActiveSupport::TestCase
       ActionController::Parameters.new(person: {}).require(:person)
     end
   end
+
+  test "require array of params" do
+    safe_params = ActionController::Parameters.new(person: {first_name: 'Gaurish', title: 'Mjallo'})
+      .require(:person)
+      .require([:first_name, :last_name])
+      assert_kind_of Array, safe_params
+      assert_equal ['Gaurish', 'Mjallo'], safe_params
```

Comment:
The asserts are indented by two spaces more than they should. Let's also add a new line after the last require.